keps/sig-node/20190717-seccomp-ga.md

```diff
@@ -387,9 +481,228 @@ alpha](#alternatives).
 
 _This section is excluded, as it is the subject of the entire proposal._
 
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable, can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/20190731-production-readiness-review-process.md
+
+Production readiness review questionnaire must be completed for features in
+v1.19 or later, but is non-blocking at this time. That is, approval is not
+required in order to be in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+
+-->
+
+### Feature enablement and rollback
+
+_This section must be completed when targeting alpha to a release._
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  
+  This feature was implemented prior to the concept of feature gates were adopted 
+  within Kubernetes. This KEP is simply "cleaning up" the API to make it GA.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we rollback
+  the enablement)?**
+  Also set `rollback-supported` to `true` or `false` in `kep.yaml`.
+  Describe the consequences on existing workloads (e.g. if this is runtime
+  feature, can it break the existing applications?).
+
+  This feature has no `rollback support`. The impact on existing workloads is defined 
+  under [Version Skew Strategy](#version-skew-strategy).
```

Comment:
I would clarify that the feature (seccomp) cannot be rolled back, but the API supports rollback of the kubernetes apiserver as described in the version skew strategy.

Reply:
I made some changes to it, please check whether it is clear enough.